glusterd2/store/store.go

```diff
@@ -177,3 +181,37 @@ func newNamespacedStore(oc *clientv3.Client, conf *Config) (*GDStore, error) {
 
 	return &GDStore{*conf, kv, lease, watcher, session, oc, nil, namespaceKey}, nil
 }
+
+//Get wrapper function to call store Get operation with timeout if context is context.TODO()
```

Comment:
The comment for these can be better:

`Get is a wrapper function that calls clientv3.KV.Get with a default timeout if an empty context is passed`

You can do the same for `Put` and `Delete`

Reply:
done